templates/user.tmpl

```diff
@@ -0,0 +1,208 @@
+                <div class="buttons" style="display: block">
+                    {{if eq .User.DeletedBy.UserID -1}}
+                        <a class="button is-info is-outlined" onclick="resetPasswordModal()">
+                            Reset password
+                        </a>
+                        <a class="button is-info is-outlined" onclick="editUserModal()">
+                            Edit
+                        </a>
+                        {{if .User.Enabled}}
+                            <a class="button is-warning is-outlined" onclick="disableUserModal()">
+                                Disable
+                            </a>
+                        {{else}}
+                            <a class="button is-warning is-outlined" onclick="enableUserModal()">
+                                Enable
+                            </a>
+                        {{end}}
+                        <a class="button is-danger is-outlined" onclick="deleteUserModal()">
+                            Delete
+                        </a>
+                    {{end}}
+                </div>
+            </div>
+            <div class="column">
+                <p id="message" style="color: green"></p>
+                <p id="error" style="color: red"></p>
+                {{parseUserIntoHTML .User .UserPermissions}}
+            </div>
+        </div>
+    </div>
+    {{if not .User.DeletedAt.Valid}}
+        {{template "modals" .}}
+    {{end}}
+{{end}}
+
+{{define "modals"}}
+    <div id="resetPasswordModal" class="modal">
+        <div class="modal-background"></div>
+        <div class="modal-content">
+            <div class="box">
+                <article class="media">
+                    <div class="media-content">
+                        <div class="content">
+                            <p class="title">Are you sure you want to reset the password?</p>
+                            <p><strong>This action cannot be undone</strong><br>An email will be sent to the user for
+                                them to reset their password</p>
+                            <a class="button is-danger" onclick="resetPassword()">Reset password</a>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+        <button class="modal-close is-large" aria-label="close"></button>
+    </div>
+    <div id="editUserModal" class="modal">
+        <div class="modal-background"></div>
+        <div class="modal-content">
+            <div class="box">
+                <article class="media">
+                    <div class="media-content">
+                        <div class="content">
+                            <p class="title">Are you sure you want to reset the password?</p>
+                            <p><strong>This action cannot be undone</strong><br>An email will be sent to the user for
+                                them to reset their password</p>
```

Comment:
Copy-paste?

Reply:
I don't think so? Which part?

Reply:
This is the editUser form, but this message looks like it belongs in resetPassword.

Reply:
Ah so it is, sorry I missed that part

Reply:
Fixed